src/Build.UnitTests/ProjectCache/ProjectCacheTests.cs

```diff
@@ -446,7 +443,7 @@ public static IEnumerable<object[]> SuccessfulGraphsWithBuildParameters
                         yield return new object[]
                         {
                             graph,
-                            ((BuildParameters) buildParameters.First()).Clone()
```

Comment:
Ok to drop the clone?

Reply:
Yup, `MultiProcWithAndWithoutInProcNode` yield returns new objects every time, so no need to clone this.